### DISCUSSION TITLE
How to output the light curve of the model

### DISCUSSION THREAD

**Discussion by AstroJLin** [phoebe-project/phoebe2#651] [Questions]
Hi,

I want to get the light curve of the model from the phoebe and draw the light curve of the model by myself, what command should I use to get light curve of model?

Thanks!

**Comment by kecnry** [comment]
After you call `run_compute` the resulting arrays are stored in the appropriate "model" (with `context='model'`) and can be accessed as any other parameter.  See the bottom of the [tutorial on computing forward models](http://phoebe-project.org/docs/latest/tutorials/compute) for details.
